src/plugins/visualizations/public/embeddable/embeddable_error_label.tsx

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
why are we implemeting this only for visualize embeddable ? looks like something every embeddable would benefit from ?

Reply:
I'm already exploring this.
But it's not that straight forward as this one.
I'm planning to extract the label code to the top level embeddable and them ask the teams to do help.

Reply:
yeah, lets make sure the embeddables supports this and then lets leave it to every author of embeddable implementations to add support for it to their specific embeddable. 

Can we please make it in this PR (making a general embeddable solution for this and then implementing the necessary inside visualize embeddable, which can then serve as an example for other teams)

Reply:
Yep, already have it ready :-P